src/server/session.ts

```diff
@@ -790,6 +820,11 @@ module ts.server {
                                                            completionDetailsArgs.entryNames,completionDetailsArgs.file);
                         break;
                     }
+                    case CommandNames.SignatureHelp: {

```

Comment:
Put the comment on the wrong PR. so putting it here again :)

Can you also update client.ts and add a test for signature help in tests\cases\fourslash\server. I would copy a signature help test from tests\cases\fourslash to start.